### DISCUSSION TITLE
Exclude Nvidia drivers on install?

### DISCUSSION THREAD

**Discussion by ZviAdzz** [prasanthrangan/hyprdots#2064] [Q&A] [answered]
I've been daily driving these for quite a while now, definitely my favorite way to use linux (on linux for 5 years +-), BUUUTTT, it bugs me a bit that they come with a driver ready to install, I have my own cachyos config for drivers that i need re-implement any time i make any changes or refresh. But, I'll also be buying a 100% AMD computer in the coming weeks and I want to run it with these dots as well, is there a simple way to exclude the nvidia drivers from installing when i run the install / refresh script?

**Comment by kRHYME7** [comment, marked as answer]
edit the script. or try the maintained one
https://github.com/Hyde-project/hyde

**Reply by ZviAdzz** [reply]
if i comment the nvidia detection will it be ok (i'm a bash noob)?

**Reply by kRHYME7** [reply]
yeah it will just skip nvidia.

> try the maintained one
https://github.com/Hyde-project/hyde


about this one we added ` ./install.sh -n ` to skip nvidia
https://github.com/HyDE-Project/HyDE/wiki/installation#modes

**Reply by ZviAdzz** [reply]
That's awesome, i'll try to re-install if when i get back home
